### DISCUSSION TITLE
How I can add a post via rest api?

### DISCUSSION THREAD

**Discussion by sakibhasandev** [microfeed/microfeed#73] [Q&A] [answered]
Hi, Is there anyway I can use rest api to delete and add a post from another client site?

**Comment by wenbinf** [comment, marked as answer]
No today.
but It's on the roadmap :)

**Comment by wenbinf** [comment]
The API is here: https://www.microfeed.org/i/new-release-v012-introducing-microfeeds-powerf-ZQxgQnG71PC/

Please sync & redeploy your microfeed instance to try it out.
